packages/page-referenda/src/Referenda/Referendum.tsx

```diff
@@ -141,13 +141,16 @@ function getChartResult (totalEligible: BN, isConvictionVote: boolean, info: Pal
         supx = (supn || supx !== -1) ? supx : i;
       }
 
-      const step = x[1].sub(x[0]);
```

Comment:
Can't really do that, because on short enough timeframes, `x[0] == x[1]` and `step` becomes zero.